src/main/java/htsjdk/samtools/cram/structure/block/Block.java

```diff
@@ -0,0 +1,234 @@
+/**
```

Comment:
Block is now the abstract base class for the 2 concrete implementations `RawBlock` and `CompressibleBlock`